ms-common/src/main/java/net/trajano/ms/core/JwtClaimsSetPrincipal.java

```diff
@@ -13,34 +17,68 @@
 public class JwtClaimsSetPrincipal implements
     Principal {
 
+    /**
+     * This is a framework specific claim which lists the roles in a String[].
+     */
+    public static final String ROLES = "roles";
+
     private final String authority;
 
-    private final JWTClaimsSet claimsSet;
+    private final JwtClaims claimsSet;
 
-    public JwtClaimsSetPrincipal(final JWTClaimsSet claimsSet) {
+    /**
+     * Roles.
+     */
+    private final Set<String> roles;
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename field "roles" to prevent any misunderstanding/clash with field "ROLES" defined on line 23. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1845)